assets/js/order-script.js

```diff
@@ -27,49 +30,49 @@ jQuery(document).ready(function($) {
 
 			window.open(url,'_blank');
 		}
-	});
+	} );
 
-	$('#wpo_wcpdf-data-input-box').insertAfter('#woocommerce-order-data');
+	$( '#wpo_wcpdf-data-input-box' ).insertAfter('#woocommerce-order-data');
 	
 	// enable invoice number edit if user initiated
-	$( ".wpo-wcpdf-set-date-number, .wpo-wcpdf-edit-date-number, .wpo-wcpdf-edit-document-notes" ).on("click", function() {
-		$form = $(this).closest('.wcpdf-data-fields');
+	$( document ).on( 'click', '.wpo-wcpdf-set-date-number, .wpo-wcpdf-edit-date-number, .wpo-wcpdf-edit-document-notes', function() {
+		let form = $(this).closest('.wcpdf-data-fields');
 
 		// check visibility
-		if( $form.find(".read-only").is(":visible") ) {
-			$form.find(".read-only").hide();
-			$form.find(".editable").show();
-			$form.find(':input').attr('disabled', false);
-			$form.closest('.wcpdf-data-fields').find('.wpo-wcpdf-document-buttons').show();
+		if( form.find(".read-only").is(":visible") ) {
+			form.find(".read-only").hide();
+			form.find(".editable").show();
+			form.find(':input').attr('disabled', false);
+			form.closest('.wcpdf-data-fields').find('.wpo-wcpdf-document-buttons').show();
 		} else {
-			$form.find(".read-only").show();
-			$form.find(".editable").hide();
-			$form.find(':input').attr('disabled', true);
-			$form.closest('.wcpdf-data-fields').find('.wpo-wcpdf-document-buttons').hide();
+			form.find(".read-only").show();
+			form.find(".editable").hide();
+			form.find(':input').attr('disabled', true);
+			form.closest('.wcpdf-data-fields').find('.wpo-wcpdf-document-buttons').hide();
 		}
-	});
+	} );
 
 	// delete document
-	$( ".wcpdf-data-fields .wpo-wcpdf-delete-document" ).on("click", function() {
+	$( document ).on( 'click', '.wcpdf-data-fields .wpo-wcpdf-delete-document', function() {
```

Comment:
Is there a performance difference? Normally you'd use this approach for elements that are dynamically added (e.g. not available in the DOM at page load). I don't think that's the case here. That said, I think you can also limit the scope of the trigger bind to a closer parent (available at page load)

Reply:
Because after save I load the document data again with jQuery.

Reply:
In that case I would still limit the scope to a closer parent for better performance